lib/db/lowlevel.go

```diff
@@ -82,6 +86,14 @@ func NewLowlevel(backend backend.Backend, opts ...Option) *Lowlevel {
 	}
 	db.keyer = newDefaultKeyer(db.folderIdx, db.deviceIdx)
 	db.Add(util.AsService(db.gcRunner, "db.Lowlevel/gcRunner"))
+	if loc := backend.Location(); loc != "" {
+		path := filepath.Join(loc, needsRepairFile)
```

Comment:
We should probably not assume that `loc` is a directory that we can do stuff in. We should be able to tack on an extension to loc though